cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -26,6 +26,9 @@ runtime-spec-compatible `dest/config.json`.
 **--type**
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
+**--log-level**
+  Log level to create runtime bundle. To be one of panic, fatal, error, warn, info, or debug. If unset, the level is identical to info.
```

Comment:
@wking 
I've researched runtime tool repo, but found it has only debug messages, without info message(so far) need feed back to consumers.
But for image tools repo, there are messages, like https://github.com/opencontainers/image-tools/pull/46/commits/5beb0201b7ffdb41f150d6f20d8471e27bc8bd37#diff-da07f373046a0a24e67b18699dc15a70R88 and https://github.com/opencontainers/image-tools/pull/46/commits/5beb0201b7ffdb41f150d6f20d8471e27bc8bd37#diff-8c24ca9b63b3598ef3f4cc64741f433eR117, need be shown to consumers.

Later we maybe add more information, which consumer does care about, so I think default info level is better.
I'm not persistent to it, but could you please consider it? Thanks.